### ISSUE TITLE
Image support

### ISSUE THREAD

**Issue by thatLeaflet** [wbernard/Memorado#70]
I wanted to use Memorado for memorizing some math things, but there is no elegant way to display math in the app's current state. Image support would be very useful for this scenario. LaTeX could also work, but that's more complicated to use and more specialized.

**Comment by MrFinnever** [comment]
Yeah, I want that feature too. That would be awesome and really useful for everyone

**Comment by XanderLeaDaren** [comment]
I’d switch to Memorado from Anki with this feature.

As a teacher, I spend the first days of each schoolyear memorizing the names of my students. Each flashcard is a student pic, the back is their name.
